packages/shared/src/components/modals/PostToSquadModal.tsx

```diff
@@ -79,13 +77,20 @@ function PostToSquadModal({
     },
   );
 
-  const onSubmit: SubmitSharePostFunc = async (e, commentary, postLink) => {
+  const onSubmit: SubmitSharePostFunc = async (e, commentary) => {
     e?.preventDefault();
 
     if (isLoading) return null;
 
-    if (isLink) {
-      return onSubmitLink({ url: postLink, sourceId: squad.id, commentary });
+    if (form?.privateLink) {
```

Comment:
Got a bit confused as to why this privateLink has multiple fields, but maybe it's just the naming to me personally.